assets/js/modules/analytics/datastore/profiles.js

```diff
@@ -188,13 +173,17 @@ export const actions = {
 };
 
 export const controls = {
-	[ FETCH_CREATE_PROFILE ]: ( { payload: { accountID, propertyID } } ) => {
+	[ FETCH_CREATE_PROFILE ]: ( { payload: { propertyID } } ) => {
+		const { accountID } = parsePropertyID( propertyID );
```

Comment:
This _could_ theoretically fail. You're catching a possible invalid `propertyID` before in `createProfile` and `fetchProfiles`, so it shouldn't be an issue. I did it slightly differently for #1403, returning an API-like error from the control. I think that approach looks less elegant, but still works seamlessly and I'd say it is a tiny bit more bulletproof (e.g. if `parsePropertyID` changed to not only be dependent on `isValidPropertyID`). But I'm okay with the approach you took here, so let's proceed.